apps/space/core/hooks/use-intersection-observer.tsx

```diff
@@ -18,7 +18,9 @@ export const useIntersectionObserver = (
       const observer = new IntersectionObserver(
         (entries) => {
           if (entries[entries.length - 1].isIntersecting) {
-            callback && callback();
+            if (callback) {
+              callback();
+            }
```

Comment:
callback?.()

Reply:
i'll keep that in mind for future lint fixes and see if our rules permit it